src/frontend/components/StudentLiveControlButtons/StudentHideAppsButton/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Button } from 'components/Button';
+import { AppsSVG } from 'components/SVGIcons/AppsSVG';
+import { useLivePanelState } from 'data/stores/useLivePanelState';
+
+const messages = defineMessages({
+  apps: {
+    defaultMessage: 'Apps',
+    description: 'Label for the hide apps button',
+    id: 'components.StudentHideAppsButton.apps',
+  },
+  hideApps: {
+    defaultMessage: 'Hide apps',
+    description: 'Title for the hide apps button',
+    id: 'components.StudentHideAppsButton.hideApps',
+  },
+});
+
+export const StudentHideAppsButton = () => {
+  const intl = useIntl();
+  const { openPanel } = useLivePanelState((state) => ({
+    openPanel: state.setPanelVisibility,
```

Comment:
IMO you should keep `setPanelVisibility` instead of `openPanel`. `openPanel` is an action and I didn't expect to call it with a boolean as argument

Reply:
https://github.com/openfun/marsha/pull/1249/commits/ab19218829bf3c8df654259c4a540c19bd6b7051